Dockerfile.alpine

```diff
@@ -45,7 +47,8 @@ WORKDIR /var/www/html
 
 COPY docker/docker.env /var/www/html/.env
 
-RUN chown -R apache:apache /var/www/html
+RUN mkdir -p "/var/www/html/vendor" \
```

Comment:
The existing Alpine Dockerfile doesn't do this, and Composer seems to still work OK. Is your thinking here that we want to pre-create the directory so the following `chown` will make the `./vendor` directory owned by Apache? Or is there some other reason?

Reply:
Sorry, while I was making the changes the composer presented a permission error when trying to change this folder, but apparently it was a storage problem in the  Docker VM. I did a rebuild without this change and it worked fine. I will make a commit correcting this.